robot/2023/src/main/java/frc/robot/Robot.java

```diff
@@ -104,18 +115,46 @@ public void disabledInit() {}
   @Override
   public void disabledPeriodic() {}
 
-  /** This autonomous runs the autonomous command selected by your {@link RobotContainer} class. */
   @Override
-  public void autonomousInit() {
-    // schedule the autonomous command (example)
-    if (m_autonomousCommand != null) {
-      m_autonomousCommand.schedule();
-    }
+  public void autonomousInit() 
+  {
+ 
+    //set robot starting distance to 0
+    leftEncoder.reset();
+    rightEncoder.reset();
+    leftMotorControllerOne.setInverted(true);
+    leftMotorControllerTwo.setInverted(true);
+    //I assume pathArray will need a value soon
+    ArrayList<Path> pathArray = new ArrayList<Path>();
+    pathElements = pathArray.iterator();
   }
 
   /** This function is called periodically during autonomous. */
   @Override
-  public void autonomousPeriodic() {}
+  public void autonomousPeriodic() 
+  {
+    //If 2 feet haven't been travelled, move forward
```

Comment:
This bit is unnecessary if Path's tick() is implemented.

Reply:
I was aware, though intended on keeping them for recording our encoder's distance per pulse.  I assumed it would have more consistent results than a human driver but I suppose the program for that is averaging them anyways.

Reply:
Keeping it now for the purpose of measurement

Reply:
It was deleted